cruise.cpp

```diff
@@ -0,0 +1,22 @@
+#include "cruise.h"
```

Comment:
This class is extinct :( I think you may have added it accidentally from an old branch